### PR TITLE
BZ2040707: Style fixes on ZTP assembly

### PR DESCRIPTION
**Pull request by cherraz**
Quite a lot of small style changes across the ZTP assembly

- Removed duplication on callouts
- Hyphenated "bare metal" when used as an adjective
- Used "bare-metal host" for consistency
- Changed the name of the assembly in the file header when needed (I think the assembly was renamed at some point, and it frustrated me not being able to know where a file was included)
- Added Jupiter content types
- Changed variables to be like "<variable_name>"  when required

Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=2040707

Preview:
*  Link to the assembly:

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#provisioning-edge-sites-at-scale_ztp-deploying-disconnected

* Links to most heavily changed  modules:

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-creating-siteconfig-custom-resources_ztp-deploying-disconnected

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-configuring-a-static-ip_ztp-deploying-disconnected

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-preparing-the-hub-cluster-for-ztp_ztp-deploying-disconnected

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-configuring-ipv6_ztp-deploying-disconnected

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-configuring-the-cluster-for-a-disconnected-environment_ztp-deploying-disconnected

https://deploy-preview-41327--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected#ztp-preparing-the-hub-cluster-for-ztp_ztp-deploying-disconnected

Versions:  4.9, 4.10

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
Only one question here, LGTM otherwise. There are lots of little changes that are tedious to track down in the previews.

File: modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc [hunk 1]
```diff
@@ -2,6 +2,7 @@
 //
 // scalability_and_performance/ztp-deploying-disconnected.adoc
 
+:_content-type: PROCEDURE
 [id="enabling-assisted-installer-service-on-bare-metal_{context}"]
 = Enabling assisted installer service on bare metal
```
[the remark this answers is not in the document]
Comment (cherraz) [reply modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc:7]: In this case it's not an adjetive, is it?

[the remark this answers is not in the document]
Comment (skrthomas) [reply modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc:7]: Ah yea not an adjective

**Comment by cmyster** [comment]
LGTM

**Pull request merged**